extensions/amp-selector/1.0/selector.type.js

```diff
@@ -34,35 +35,26 @@ var SelectorDef = {};
  */
 SelectorDef.Props;
 
-/**
- * @typedef {{
- *   domElement: !Element,
- *   disabled: (boolean|undefined),
- *   multiple: (boolean|undefined),
- *   role: string,
- * }}
- */
-SelectorDef.ShimProps;
-
 /**
  * @typedef {{
  *   as: (string|PreactDef.FunctionalComponent),
  *   option: *,
  *   disabled: (boolean|undefined),
+ *   domElement: !Element,
  *   onClick: (?function(!Event)|undefined),
  *   role: string,
+ *   selected: (boolean|undefined),
```

Comment:
This is also strictly for shimming? I still have some concerns about it. Maybe we should do either `_domElement` and `_selected` to indicate that these are private props. Prefix or suffix ok. Or maybe even more explicitly `shimDomElement` and `shimSelected`?